### DISCUSSION TITLE
Mobile API

### DISCUSSION THREAD

**Discussion by StarkSoftware** [orchidsoftware/platform#1650] [General]
I wanted to know it was possible to connect a mobile app via API.

**Comment by kubre** [comment]
In short Yes, It's possible.
Though as per my knowledge orchid does not currently offer any way to build the api's.
But Laravel itself offers ability to build api's easily, check out [Eloquent Resources in Laravel](https://laravel.com/docs/8.x/eloquent-resources#introduction).
